meta-xt-control-domain-virtio/conf/layer.conf

```diff
@@ -9,5 +9,5 @@ BBFILE_COLLECTIONS += "xt-control-domain-virtio"
 BBFILE_PATTERN_xt-control-domain-virtio := "^${LAYERDIR}/"
 BBFILE_PRIORITY_xt-control-domain-virtio = "8"
 
-LAYERSERIES_COMPAT_xt-control-domain-virtio = "kirkstone"
+LAYERSERIES_COMPAT_xt-control-domain-virtio = "scarthgap"
```

Comment:
I believe we need both options. I.e. LAYERSERIES_COMPAT_xt-control-domain-virtio = "scarthgap kirkstone"

Reply:
It should be checked.